### ISSUE TITLE
how can i finetuning pretrained model with my own audiofiles? 

### ISSUE THREAD

**Issue by MuruganR96** [facebookresearch/wav2letter#93]
how can i finetuning pretrained model with my own audiofiles? 
i was tried to trained my own audiofile and generates model(it takes huge time and results also improper) but i don't know how to finetune our pretrained model?

give you some hint sir? thank you for advance.

**Comment by murugancmi** [comment]
@MuruganR96 have a look this opensource ASR from Mozila more accuracy and good DOC,Pre trained model https://github.com/mozilla/DeepSpeech

**Comment by MuruganR96** [comment]
Thank you sir.
